src/core/Core.cpp

```diff
@@ -752,10 +752,20 @@ void ManageCombatModeAnimations() {
 			if(layer1.cur_anim == alist[ANIM_MISSILE_WAIT]) {
 				player.m_aimTime = PlatformDuration::ofRaw(1);
 				
-				if(eeMousePressed1() && getInventoryItemWithLowestDurability("arrows", 1.f) != nullptr) {
-					changeAnimation(io, 1, alist[ANIM_MISSILE_STRIKE_PART_1]);
-					io->isHit = false;
+				if(eeMousePressed1()) {
+					if (getInventoryItemWithLowestDurability("arrow", 1.f) != nullptr) {
```

Comment:
you could put these 2 conditions into an OR as the bodies of the blocks seem to be the same?